lib/utils/test_utils_test.go

```diff
@@ -22,20 +22,6 @@ func TestNewTestDir(t *testing.T) {
 	RemoveTestDir(t)
 }
 
-// TestNewTestBasePath tests the NewTestBasePath method
-func TestNewTestBasePath(t *testing.T) {
-	basePath := "test"
-
-	testDir := NewTestBasePath(t, basePath)
-
-	expected := path.Join(TestDir, t.Name(), basePath)
-
-	require.Equal(t, expected, testDir)
-	require.Equal(t, PathExists(testDir), true)
-
-	RemoveTestDir(t)
-}
```

Comment:
Why removing this?

Reply:
because this Test tests if the `NewTestBasePath` (which is a helper test) so I think this is not useful

Reply:
And I modified the `NewTestBasePath` to use `t.TmpDir()` instead of create a `./test_data` file in the current folder to test things

Reply:
Awesome! By the way feel free to review my start-of-the-year #2137 :smile: